Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift

```diff
@@ -88,7 +88,11 @@ internal class CLDNetworkCoordinator {
         else if let apiSecret = config.apiSecret {
             let timestamp = Int(Date().timeIntervalSince1970)
             params[CLDSignature.SignatureParam.Timestamp.rawValue] = cldParamValueAsString(value: timestamp)
-            let signature = cloudinarySignParamsUsingSecret(params, cloudinaryApiSecret: apiSecret)
+            
+            var paramsWithoutTimeout = params
```

Comment:
Instead of removing the timeout param directly here, please add a private method that strips all the non-signature params (so just timeout, for now) and call it from here.

It'll make the method clearer.